models/model/core/v1alpha1/component.go

```diff
@@ -42,7 +42,7 @@ type ComponentDefinition struct {
 	UpdatedAt       time.Time              `json:"-"`
 }
 type ComponentDefinitionDB struct {
-	ID      uuid.UUID `json:"id"`
+	ID      uuid.UUID `json:"componentDefinitionId"`
```

Comment:
@MUzairS15 @leecalcote  To fix the golangci-lint issue, I renamed the Json tag. Will it have an impact on its dependencies?

Reply:
@Yashsharma1911 Could confirm.

Reply:
@Yashsharma1911 may be slow to response, therefore I revert this unpredictable modification so that we can keep going. // @MUzairS15 @leecalcote

Reply:
@RipulHandoo, I wonder if you might be familiar here.

Reply:
@leecalcote @MUzairS15 Can we go ahead, and I can create another issue mentioning @Yashsharma1911 @RipulHandoo to track it?

Reply:
The issue created: https://github.com/meshery/meshkit/issues/469